day8/README.md

```diff
@@ -0,0 +1,68 @@
+Question: #2 from this [CS170 discussion](https://d1b10bmlvqabco.cloudfront.net/attach/ixmn3a7kjp7186/hktxrll0pr53jq/iz0mn2w8l3du/sec4sol.pdf)
+
+Alien Alphabet
+
+Suppose you have a dictionary of an alien language which
+lists words in some sorted lexicographical ordering. For
+example, given the following list of words:
+
+`[baa abcd abca cab cad]`
+
+You can conclude the ordering of the alphabet is
+
+`b < d < a < c`
+
+Write a program to determine the lexicographical
+ordering for any input list.
+
+## Ideas
+
+This alphabet ordering reminds me of depency graphs. Based
+on the words list, some letters come before other letters.
+Some letters have multiple letters that precede it in the 
+words list. The trick is to find how those letters relate to
+each other.
+
+If I can represent the words/letters as a dependency graph,
```

Comment:
yes!! this is so exciting, I'm really excited that I recognized what this problem was about HAHA